[checkstyle] Resolve issues

`checkstyle` reports the following unresolved issues as of `1.13.2`

```
2017-10-13T18:11:23.2486689Z :adal:checkstyle[ant:checkstyle] E:\ag\_work\25\s\adal\src\androidTest\java\com\microsoft\aad\adal\AcquireTokenRequestTest.java:336:72: warning: ',' is not followed by whitespace.
2017-10-13T18:11:23.2486689Z [ant:checkstyle] E:\ag\_work\25\s\adal\src\androidTest\java\com\microsoft\aad\adal\AuthenticationActivityUnitTest.java:93:59: warning: Variable 'mActivityRule' must be private and have accessor methods.
2017-10-13T18:11:23.2486689Z [ant:checkstyle] E:\ag\_work\25\s\adal\src\androidTest\java\com\microsoft\aad\adal\AuthenticationContextTest.java:38:8: warning: Unused import - android.support.v4.hardware.fingerprint.FingerprintManagerCompat.
2017-10-13T18:11:23.2486689Z [ant:checkstyle] E:\ag\_work\25\s\adal\src\androidTest\java\com\microsoft\aad\adal\AuthenticationContextTest.java:1266:48: warning: '3' is a magic number.
2017-10-13T18:11:23.2486689Z [ant:checkstyle] E:\ag\_work\25\s\adal\src\androidTest\java\com\microsoft\aad\adal\AuthenticationContextTest.java:1345:48: warning: '3' is a magic number.
2017-10-13T18:11:23.2486689Z [ant:checkstyle] E:\ag\_work\25\s\adal\src\androidTest\java\com\microsoft\aad\adal\AuthenticationContextTest.java:1478:47: warning: '8' is a magic number.
2017-10-13T18:11:23.2486689Z [ant:checkstyle] E:\ag\_work\25\s\adal\src\androidTest\java\com\microsoft\aad\adal\AuthorityValidationMetadataCacheTest.java:65:60: warning: '5' is a magic number.
2017-10-13T18:11:23.2486689Z [ant:checkstyle] E:\ag\_work\25\s\adal\src\androidTest\java\com\microsoft\aad\adal\AuthorityValidationMetadataCacheTest.java:99:37: warning: '+' should be on a new line.
2017-10-13T18:11:23.2486689Z [ant:checkstyle] E:\ag\_work\25\s\adal\src\androidTest\java\com\microsoft\aad\adal\AuthorityValidationMetadataCacheTest.java:100:21: warning: '+' should be on a new line.
2017-10-13T18:11:23.2486689Z [ant:checkstyle] E:\ag\_work\25\s\adal\src\androidTest\java\com\microsoft\aad\adal\AuthorityValidationMetadataCacheTest.java:101:73: warning: '+' should be on a new line.
2017-10-13T18:11:23.2486689Z [ant:checkstyle] E:\ag\_work\25\s\adal\src\androidTest\java\com\microsoft\aad\adal\AuthorityValidationMetadataCacheTest.java:102:63: warning: '+' should be on a new line.
2017-10-13T18:11:23.2486689Z [ant:checkstyle] E:\ag\_work\25\s\adal\src\androidTest\java\com\microsoft\aad\adal\AuthorityValidationMetadataCacheTest.java:103:34: warning: '+' should be on a new line.
2017-10-13T18:11:23.2486689Z [ant:checkstyle] E:\ag\_work\25\s\adal\src\androidTest\java\com\microsoft\aad\adal\AuthorityValidationMetadataCacheTest.java:104:50: warning: '+' should be on a new line.
2017-10-13T18:11:23.2486689Z [ant:checkstyle] E:\ag\_work\25\s\adal\src\androidTest\java\com\microsoft\aad\adal\AuthorityValidationMetadataCacheTest.java:105:42: warning: '+' should be on a new line.
2017-10-13T18:11:23.2486689Z [ant:checkstyle] E:\ag\_work\25\s\adal\src\androidTest\java\com\microsoft\aad\adal\AuthorityValidationMetadataCacheTest.java:106:44: warning: '+' should be on a new line.
2017-10-13T18:11:23.2486689Z [ant:checkstyle] E:\ag\_work\25\s\adal\src\androidTest\java\com\microsoft\aad\adal\AuthorityValidationMetadataCacheTest.java:107:41: warning: '+' should be on a new line.
2017-10-13T18:11:23.2486689Z [ant:checkstyle] E:\ag\_work\25\s\adal\src\androidTest\java\com\microsoft\aad\adal\AuthorityValidationMetadataCacheTest.java:108:21: warning: '+' should be on a new line.
2017-10-13T18:11:23.2486689Z [ant:checkstyle] E:\ag\_work\25\s\adal\src\androidTest\java\com\microsoft\aad\adal\AuthorityValidationMetadataCacheTest.java:109:22: warning: '+' should be on a new line.
2017-10-13T18:11:23.2486689Z [ant:checkstyle] E:\ag\_work\25\s\adal\src\androidTest\java\com\microsoft\aad\adal\AuthorityValidationMetadataCacheTest.java:110:21: warning: '+' should be on a new line.
2017-10-13T18:11:23.2486689Z [ant:checkstyle] E:\ag\_work\25\s\adal\src\androidTest\java\com\microsoft\aad\adal\AuthorityValidationMetadataCacheTest.java:111:72: warning: '+' should be on a new line.
2017-10-13T18:11:23.2486689Z [ant:checkstyle] E:\ag\_work\25\s\adal\src\androidTest\java\com\microsoft\aad\adal\AuthorityValidationMetadataCacheTest.java:112:70: warning: '+' should be on a new line.
2017-10-13T18:11:23.2486689Z [ant:checkstyle] E:\ag\_work\25\s\adal\src\androidTest\java\com\microsoft\aad\adal\AuthorityValidationMetadataCacheTest.java:113:34: warning: '+' should be on a new line.
2017-10-13T18:11:23.2486689Z [ant:checkstyle] E:\ag\_work\25\s\adal\src\androidTest\java\com\microsoft\aad\adal\AuthorityValidationMetadataCacheTest.java:114:48: warning: '+' should be on a new line.
2017-10-13T18:11:23.2486689Z [ant:checkstyle] E:\ag\_work\25\s\adal\src\androidTest\java\com\microsoft\aad\adal\AuthorityValidationMetadataCacheTest.java:115:21: warning: '+' should be on a new line.
2017-10-13T18:11:23.2486689Z [ant:checkstyle] E:\ag\_work\25\s\adal\src\androidTest\java\com\microsoft\aad\adal\AuthorityValidationMetadataCacheTest.java:116:21: warning: '+' should be on a new line.
2017-10-13T18:11:23.2486689Z [ant:checkstyle] E:\ag\_work\25\s\adal\src\androidTest\java\com\microsoft\aad\adal\BrokerAccountServiceTest.java:76:28: warning: Variable 'mServiceTestRule' must be private and have accessor methods.
2017-10-13T18:11:23.2486689Z [ant:checkstyle] E:\ag\_work\25\s\adal\src\androidTest\java\com\microsoft\aad\adal\StorageHelperTests.java:295:17: warning: Name 'testKeyPair_AndroidKeyStore' must match pattern '^[a-z][a-zA-Z0-9]*$'.
2017-10-13T18:11:23.2486689Z [ant:checkstyle] E:\ag\_work\25\s\adal\src\androidTest\java\com\microsoft\aad\adal\Util.java:140:142: warning: '+' should be on a new line.
2017-10-13T18:11:23.2486689Z [ant:checkstyle] E:\ag\_work\25\s\adal\src\androidTest\java\com\microsoft\aad\adal\Util.java:141:68: warning: '+' should be on a new line.
2017-10-13T18:11:23.2486689Z [ant:checkstyle] E:\ag\_work\25\s\adal\src\androidTest\java\com\microsoft\aad\adal\Util.java:142:74: warning: '+' should be on a new line.
2017-10-13T18:11:23.2486689Z [ant:checkstyle] E:\ag\_work\25\s\adal\src\main\java\com\microsoft\aad\adal\AcquireTokenRequest.java:60:28: warning: Name 'mHandler' must match pattern '^[s][a-zA-Z0-9]*$'.
2017-10-13T18:11:23.2486689Z [ant:checkstyle] E:\ag\_work\25\s\adal\src\main\java\com\microsoft\aad\adal\AuthenticationConstants.java:320:9: warning: Missing a Javadoc comment.
2017-10-13T18:11:23.2486689Z [ant:checkstyle] E:\ag\_work\25\s\adal\src\main\java\com\microsoft\aad\adal\AuthenticationConstants.java:495:41: warning: Name 'sPrefix' must match pattern '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'.
2017-10-13T18:11:23.2486689Z [ant:checkstyle] E:\ag\_work\25\s\adal\src\main\java\com\microsoft\aad\adal\AuthenticationConstants.java:497:13: warning: Missing a Javadoc comment.
2017-10-13T18:11:23.2486689Z [ant:checkstyle] E:\ag\_work\25\s\adal\src\main\java\com\microsoft\aad\adal\AuthenticationConstants.java:499:13: warning: Missing a Javadoc comment.
2017-10-13T18:11:23.2486689Z [ant:checkstyle] E:\ag\_work\25\s\adal\src\main\java\com\microsoft\aad\adal\AuthenticationConstants.java:501:13: warning: Missing a Javadoc comment.
2017-10-13T18:11:23.2486689Z [ant:checkstyle] E:\ag\_work\25\s\adal\src\main\java\com\microsoft\aad\adal\AuthenticationConstants.java:503:13: warning: Missing a Javadoc comment.
2017-10-13T18:11:23.2486689Z [ant:checkstyle] E:\ag\_work\25\s\adal\src\main\java\com\microsoft\aad\adal\AuthenticationConstants.java:527: warning: First sentence should end with a period.
2017-10-13T18:11:23.2486689Z [ant:checkstyle] E:\ag\_work\25\s\adal\src\main\java\com\microsoft\aad\adal\AuthorityValidationMetadataCache.java:43:1: warning: Utility classes should not have a public or default constructor.
2017-10-13T18:11:23.2486689Z [ant:checkstyle] E:\ag\_work\25\s\adal\src\main\java\com\microsoft\aad\adal\BrokerProxy.java:366:111: warning: '{' is not preceded with whitespace.
2017-10-13T18:11:23.2486689Z [ant:checkstyle] E:\ag\_work\25\s\adal\src\main\java\com\microsoft\aad\adal\BrokerProxy.java:465:108: warning: '{' is not preceded with whitespace.
2017-10-13T18:11:23.2486689Z [ant:checkstyle] E:\ag\_work\25\s\adal\src\main\java\com\microsoft\aad\adal\DRSMetadata.java:36: warning: First sentence should end with a period.
2017-10-13T18:11:23.2486689Z [ant:checkstyle] E:\ag\_work\25\s\adal\src\main\java\com\microsoft\aad\adal\Discovery.java:70:22: warning: 'static' modifier out of order with the JLS suggestions.
2017-10-13T18:11:23.2486689Z [ant:checkstyle] E:\ag\_work\25\s\adal\src\main\java\com\microsoft\aad\adal\IdentityProviderService.java:38: warning: First sentence should end with a period.
2017-10-13T18:11:23.2486689Z [ant:checkstyle] E:\ag\_work\25\s\adal\src\main\java\com\microsoft\aad\adal\JWSBuilder.java:62: warning: Class Claims should be declared as final.
2017-10-13T18:11:23.2486689Z [ant:checkstyle] E:\ag\_work\25\s\adal\src\main\java\com\microsoft\aad\adal\JWSBuilder.java:72: warning: First sentence should end with a period.
2017-10-13T18:11:23.2486689Z [ant:checkstyle] E:\ag\_work\25\s\adal\src\main\java\com\microsoft\aad\adal\JWSBuilder.java:83: warning: Class JwsHeader should be declared as final.
2017-10-13T18:11:23.2486689Z [ant:checkstyle] E:\ag\_work\25\s\adal\src\main\java\com\microsoft\aad\adal\JWSBuilder.java:93: warning: First sentence should end with a period.
2017-10-13T18:11:23.2486689Z [ant:checkstyle] E:\ag\_work\25\s\adal\src\main\java\com\microsoft\aad\adal\Link.java:47: warning: First sentence should end with a period.
2017-10-13T18:11:23.2486689Z [ant:checkstyle] E:\ag\_work\25\s\adal\src\main\java\com\microsoft\aad\adal\PowerManagerWrapper.java:35:32: warning: Variable 'sInstance' must be private and have accessor methods.
2017-10-13T18:11:23.2486689Z [ant:checkstyle] E:\ag\_work\25\s\adal\src\main\java\com\microsoft\aad\adal\PowerManagerWrapper.java:54:51: warning: Expected @param tag for 'connectionContext'.
2017-10-13T18:11:23.2486689Z [ant:checkstyle] E:\ag\_work\25\s\adal\src\main\java\com\microsoft\aad\adal\PowerManagerWrapper.java:64:65: warning: Expected @param tag for 'connectionContext'.
2017-10-13T18:11:23.2486689Z [ant:checkstyle] E:\ag\_work\25\s\adal\src\main\java\com\microsoft\aad\adal\SSOStateSerializer.java:65: warning: First sentence should end with a period.
2017-10-13T18:11:23.2486689Z [ant:checkstyle] E:\ag\_work\25\s\adal\src\main\java\com\microsoft\aad\adal\TokenCacheAccessor.java:116:25: warning: ')' is preceded with whitespace.
2017-10-13T18:11:23.2486689Z [ant:checkstyle] E:\ag\_work\25\s\adal\src\main\java\com\microsoft\aad\adal\TokenCacheAccessor.java:233:136: warning: '{' is not preceded with whitespace.
2017-10-13T18:11:23.2486689Z [ant:checkstyle] E:\ag\_work\25\s\adal\src\main\java\com\microsoft\aad\adal\UsageStatsManagerWrapper.java:35:37: warning: Variable 'sInstance' must be private and have accessor methods.
2017-10-13T18:11:23.2486689Z [ant:checkstyle] E:\ag\_work\25\s\adal\src\main\java\com\microsoft\aad\adal\UsageStatsManagerWrapper.java:54:48: warning: Expected @param tag for 'connectionContext'.
2017-10-13T18:11:23.2486689Z [ant:checkstyle] E:\ag\_work\25\s\adal\src\main\java\com\microsoft\aad\adal\WebFingerMetadata.java:49: warning: First sentence should end with a period.
2017-10-13T18:11:23.2486689Z [ant:checkstyle] E:\ag\_work\25\s\adal\src\telemetry\java\com\microsoft\aad\adal\TelemetryUtils.java:50:32: warning: 'mVersion' hides a field.
2017-10-13T18:11:23.2486689Z [ant:checkstyle] E:\ag\_work\25\s\adal\src\telemetry\java\com\microsoft\aad\adal\TelemetryUtils.java:58:40: warning: 'mServerErrorCode' hides a field.
2017-10-13T18:11:23.2486689Z [ant:checkstyle] E:\ag\_work\25\s\adal\src\telemetry\java\com\microsoft\aad\adal\TelemetryUtils.java:66:43: warning: 'mServerSubErrorCode' hides a field.
2017-10-13T18:11:23.2486689Z [ant:checkstyle] E:\ag\_work\25\s\adal\src\telemetry\java\com\microsoft\aad\adal\TelemetryUtils.java:74:40: warning: 'mRefreshTokenAge' hides a field.
2017-10-13T18:11:23.2486689Z [ant:checkstyle] E:\ag\_work\25\s\adal\src\telemetry\java\com\microsoft\aad\adal\TelemetryUtils.java:82:32: warning: 'mSpeRing' hides a field.
```

--- a/adal/src/androidTest/java/com/microsoft/aad/adal/AcquireTokenRequestTest.java
+++ b/adal/src/androidTest/java/com/microsoft/aad/adal/AcquireTokenRequestTest.java
@@ -333,7 +333,7 @@ public final class AcquireTokenRequestTest {
     @Test
     public void testBothLocalAndBrokerSilentAuthFailedSwitchedToBrokerForInteractive()
             throws OperationCanceledException, IOException, AuthenticatorException,
-            PackageManager.NameNotFoundException, InterruptedException,JSONException {
+            PackageManager.NameNotFoundException, InterruptedException, JSONException {
 
         // Make sure AT is expired
         final ITokenCacheStore cacheStore = getTokenCache(getExpireDate(-MINUS_MINUTE), false, false, null);

--- a/adal/src/androidTest/java/com/microsoft/aad/adal/AuthenticationActivityUnitTest.java
+++ b/adal/src/androidTest/java/com/microsoft/aad/adal/AuthenticationActivityUnitTest.java
@@ -89,6 +89,7 @@ public class AuthenticationActivityUnitTest {
 
     private Intent mIntentToStartActivity;
 
+    @SuppressWarnings("checkstyle:visibilitymodifier")
     @Rule
     public final ActivityTestRule<AuthenticationActivity> mActivityRule = new ActivityTestRule<>(AuthenticationActivity.class, true, false);
 

--- a/adal/src/androidTest/java/com/microsoft/aad/adal/AuthenticationContextTest.java
+++ b/adal/src/androidTest/java/com/microsoft/aad/adal/AuthenticationContextTest.java
@@ -35,7 +35,6 @@ import android.os.Build;
 import android.os.Bundle;
 import android.support.test.InstrumentationRegistry;
 import android.support.test.runner.AndroidJUnit4;
-import android.support.v4.hardware.fingerprint.FingerprintManagerCompat;
 import android.test.UiThreadTest;
 import android.util.Base64;
 import android.util.Log;
@@ -85,7 +84,6 @@ import javax.crypto.SecretKeyFactory;
 import javax.crypto.spec.PBEKeySpec;
 import javax.crypto.spec.SecretKeySpec;
 
-import static android.R.attr.resource;
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertFalse;
 import static org.junit.Assert.assertNotNull;
@@ -1263,7 +1261,8 @@ public final class AuthenticationContextTest {
 
         latch2.await();
 
-        Mockito.verify(mockedConnection, times(3)).getInputStream();
+        final int invocationCount = 3;
+        Mockito.verify(mockedConnection, times(invocationCount)).getInputStream();
 
         // Clean up Authority validation cache
         AuthorityValidationMetadataCache.clearAuthorityValidationCache();
@@ -1342,7 +1341,8 @@ public final class AuthenticationContextTest {
 
         latch2.await();
 
-        Mockito.verify(mockedConnection, times(3)).getInputStream();
+        final int invocationCount = 3;
+        Mockito.verify(mockedConnection, times(invocationCount)).getInputStream();
     }
 
     /**
@@ -1475,7 +1475,8 @@ public final class AuthenticationContextTest {
 
         final SharedPreferences sharedPreferences =  mockContext.getSharedPreferences("com.microsoft.aad.adal.cache", Activity.MODE_PRIVATE);
         final Map<String, String> allTokens = (Map<String, String>) sharedPreferences.getAll();
-        Assert.assertTrue(allTokens.size() == 8);
+        final int expectedMapSize = 8;
+        Assert.assertTrue(allTokens.size() == expectedMapSize);
         // also verify all the key are using preferred_cache
         final Set<String> keys = allTokens.keySet();
         for (final String key : keys) {

--- a/adal/src/androidTest/java/com/microsoft/aad/adal/AuthorityValidationMetadataCacheTest.java
+++ b/adal/src/androidTest/java/com/microsoft/aad/adal/AuthorityValidationMetadataCacheTest.java
@@ -62,7 +62,9 @@ public class AuthorityValidationMetadataCacheTest extends AndroidTestHelper {
         AuthorityValidationMetadataCache.processInstanceDiscoveryMetadata(authorityURL, getDiscoveryResponse());
 
         Map<String, InstanceDiscoveryMetadata> authorityValidationMap = AuthorityValidationMetadataCache.getAuthorityValidationMetadataCache();
-        Assert.assertTrue(authorityValidationMap.size() == 5);
+
+        final int expectedMapSize = 5;
+        Assert.assertTrue(authorityValidationMap.size() == expectedMapSize);
         Assert.assertTrue(authorityValidationMap.containsKey("login.windows.net"));
         Assert.assertTrue(authorityValidationMap.containsKey("login.microsoftonline.com"));
         Assert.assertTrue(authorityValidationMap.containsKey("sts.microsoft.com"));
@@ -96,25 +98,25 @@ public class AuthorityValidationMetadataCacheTest extends AndroidTestHelper {
     }
 
     static String getMetadata() {
-        final String metadata = "[" +
-                "{" +
-                "\"preferred_network\": \"login.microsoftonline.com\"," +
-                "\"preferred_cache\": \"login.windows.net\"," +
-                "\"aliases\": [" +
-                "\"login.microsoftonline.com\"," +
-                "\"login.windows.net\"," +
-                "\"login.microsoft.com\"," +
-                "\"sts.microsoft.com\"" +
-                "]" +
-                "}," +
-                "{" +
-                "\"preferred_network\": \"login.microsoftonline.de\"," +
-                "\"preferred_cache\": \"login.microsoftonline.de\"," +
-                "\"aliases\": [" +
-                "\"login.microsoftonline.de\"" +
-                "]" +
-                "}" +
-                "]";
+        final String metadata = "["
+                + "{"
+                + "\"preferred_network\": \"login.microsoftonline.com\","
+                + "\"preferred_cache\": \"login.windows.net\","
+                + "\"aliases\": ["
+                + "\"login.microsoftonline.com\","
+                + "\"login.windows.net\","
+                + "\"login.microsoft.com\","
+                + "\"sts.microsoft.com\""
+                + "]"
+                + "},"
+                + "{"
+                + "\"preferred_network\": \"login.microsoftonline.de\","
+                + "\"preferred_cache\": \"login.microsoftonline.de\","
+                + "\"aliases\": ["
+                + "\"login.microsoftonline.de\""
+                + "]"
+                + "}"
+                + "]";
 
         return metadata;
     }

--- a/adal/src/androidTest/java/com/microsoft/aad/adal/BrokerAccountServiceTest.java
+++ b/adal/src/androidTest/java/com/microsoft/aad/adal/BrokerAccountServiceTest.java
@@ -68,10 +68,14 @@ import static org.junit.Assert.fail;
  * Test cases for brokerAccountService and related operations in {@link BrokerProxy}.
  */
 public final class BrokerAccountServiceTest {
+
     private static ExecutorService sThreadExecutor = Executors.newSingleThreadExecutor();
+
     private static final String VALID_AUTHORITY = "https://login.microsoftonline.com";
 
     private IBinder mIBinder;
+
+    @SuppressWarnings("checkstyle:visibilitymodifier")
     @Rule
     public ServiceTestRule mServiceTestRule = new ServiceTestRule();
 

--- a/adal/src/androidTest/java/com/microsoft/aad/adal/FileMockContext.java
+++ b/adal/src/androidTest/java/com/microsoft/aad/adal/FileMockContext.java
@@ -77,11 +77,11 @@ class FileMockContext extends MockContext {
         if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
             UsageStatsManagerWrapper mockUsageStatsManagerWrapper = mock(UsageStatsManagerWrapper.class);
             Mockito.when(mockUsageStatsManagerWrapper.isAppInactive(any(Context.class))).thenReturn(false);
-            UsageStatsManagerWrapper.sInstance = mockUsageStatsManagerWrapper;
+            UsageStatsManagerWrapper.setInstance(mockUsageStatsManagerWrapper);
             PowerManagerWrapper mockPowerManagerWrapper = mock(PowerManagerWrapper.class);
             Mockito.when(mockPowerManagerWrapper.isDeviceIdleMode(any(Context.class))).thenReturn(false);
             Mockito.when(mockPowerManagerWrapper.isIgnoringBatteryOptimizations(any(Context.class))).thenReturn(false);
-            PowerManagerWrapper.sInstance = mockPowerManagerWrapper;
+            PowerManagerWrapper.setInstance(mockPowerManagerWrapper);
         }
 
         // default
@@ -191,7 +191,7 @@ class FileMockContext extends MockContext {
         if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
             UsageStatsManagerWrapper mockUsageStatsManagerWrapper = mock(UsageStatsManagerWrapper.class);
             Mockito.when(mockUsageStatsManagerWrapper.isAppInactive(any(Context.class))).thenReturn(true);
-            UsageStatsManagerWrapper.sInstance = mockUsageStatsManagerWrapper;
+            UsageStatsManagerWrapper.setInstance(mockUsageStatsManagerWrapper);
         }
     }
 
@@ -200,7 +200,7 @@ class FileMockContext extends MockContext {
             PowerManagerWrapper mockPowerManagerWrapper = mock(PowerManagerWrapper.class);
             Mockito.when(mockPowerManagerWrapper.isDeviceIdleMode(any(Context.class))).thenReturn(true);
             Mockito.when(mockPowerManagerWrapper.isIgnoringBatteryOptimizations(any(Context.class))).thenReturn(false);
-            PowerManagerWrapper.sInstance = mockPowerManagerWrapper;
+            PowerManagerWrapper.setInstance(mockPowerManagerWrapper);
         }
     }
 

--- a/adal/src/androidTest/java/com/microsoft/aad/adal/StorageHelperTests.java
+++ b/adal/src/androidTest/java/com/microsoft/aad/adal/StorageHelperTests.java
@@ -292,7 +292,7 @@ public class StorageHelperTests extends AndroidTestHelper {
 
     @TargetApi(MIN_SDK_VERSION)
     @Test
-    public void testKeyPair_AndroidKeyStore() throws
+    public void testKeyPairAndroidKeyStore() throws
             GeneralSecurityException, IOException {
         if (Build.VERSION.SDK_INT < MIN_SDK_VERSION) {
             return;

--- a/adal/src/androidTest/java/com/microsoft/aad/adal/Util.java
+++ b/adal/src/androidTest/java/com/microsoft/aad/adal/Util.java
@@ -137,10 +137,13 @@ final class Util {
     static String getErrorResponseBody(final String errorCode) throws JSONException {
         final JSONObject jsonObject = new JSONObject();
         jsonObject.put(AuthenticationConstants.OAuth2.ERROR, errorCode);
-        jsonObject.put(AuthenticationConstants.OAuth2.ERROR_DESCRIPTION, "AADSTS70000: Authentication failed. Refresh Token is not valid.\n" +
-                "Trace ID: bb27293d-74e4-4390-882b-037a63429026\n" +
-                "Correlation ID: b73106d5-419b-4163-8bc6-d2c18f1b1a13\n" +
-                "Timestamp: 2014-11-06 18:39:47Z");
+        jsonObject.put(
+                AuthenticationConstants.OAuth2.ERROR_DESCRIPTION,
+                "AADSTS70000: Authentication failed. Refresh Token is not valid.\n"
+                        + "Trace ID: bb27293d-74e4-4390-882b-037a63429026\n"
+                        + "Correlation ID: b73106d5-419b-4163-8bc6-d2c18f1b1a13\n"
+                        + "Timestamp: 2014-11-06 18:39:47Z"
+        );
         jsonObject.put(AuthenticationConstants.OAuth2.ERROR_CODES, "[70000]");
         jsonObject.put("timestamp", "2014-11-06 18:39:47Z");
         jsonObject.put("trace_id", "bb27293d-74e4-4390-882b-037a63429026");

--- a/adal/src/main/java/com/microsoft/aad/adal/AcquireTokenRequest.java
+++ b/adal/src/main/java/com/microsoft/aad/adal/AcquireTokenRequest.java
@@ -57,7 +57,7 @@ class AcquireTokenRequest {
     private TokenCacheAccessor mTokenCacheAccessor;
     private final IBrokerProxy mBrokerProxy;
 
-    private static Handler mHandler = null;
+    private static Handler sHandler = null;
 
     /**
      * Instance validation related calls are serviced inside Discovery as a
@@ -797,13 +797,13 @@ class AcquireTokenRequest {
     }
 
     private synchronized Handler getHandler() {
-        if (mHandler == null) {
+        if (sHandler == null) {
             HandlerThread acquireTokenHandlerThread = new HandlerThread("AcquireTokenRequestHandlerThread");
             acquireTokenHandlerThread.start();
-            mHandler = new Handler(acquireTokenHandlerThread.getLooper());
+            sHandler = new Handler(acquireTokenHandlerThread.getLooper());
         }
 
-        return mHandler;
+        return sHandler;
     }
 
     private void waitingRequestOnError(final AuthenticationRequestState waitingRequest,

--- a/adal/src/main/java/com/microsoft/aad/adal/AuthenticationConstants.java
+++ b/adal/src/main/java/com/microsoft/aad/adal/AuthenticationConstants.java
@@ -317,6 +317,7 @@ public final class AuthenticationConstants {
         /** String of broker protocol version with PRT support. */
         public static final String BROKER_PROTOCOL_VERSION = "v2";
 
+        /** String of broker skip cache. */
         public static final String BROKER_SKIP_CACHE = "skip.cache";
 
         /** String of broker result returned. */
@@ -490,17 +491,32 @@ public final class AuthenticationConstants {
         /** String for ssl prefix. */
         public static final String REDIRECT_SSL_PREFIX = "https://";
 
+        /**
+         * Bundle identifiers for x-ms-clitelem info.
+         */
         public static final class CliTelemInfo {
 
-            private static final String sPrefix = "cliteleminfo.";
+            private static final String PREFIX = "cliteleminfo.";
 
-            public static final String SERVER_ERROR = sPrefix + "server_error";
+            /**
+             * Bundle id for server errors.
+             */
+            public static final String SERVER_ERROR = PREFIX + "server_error";
 
-            public static final String SERVER_SUBERROR = sPrefix + "server_suberror";
+            /**
+             * Bundle id for server suberrors.
+             */
+            public static final String SERVER_SUBERROR = PREFIX + "server_suberror";
 
-            public static final String RT_AGE = sPrefix + "rt_age";
+            /**
+             * Bundle id for refresh token age.
+             */
+            public static final String RT_AGE = PREFIX + "rt_age";
 
-            public static final String SPE_RING = sPrefix + "spe_ring";
+            /**
+             * Bundle id for spe_ring info.
+             */
+            public static final String SPE_RING = PREFIX + "spe_ring";
         }
     }
 
@@ -525,7 +541,7 @@ public final class AuthenticationConstants {
         static final String ACCEPT = "Accept";
 
         /**
-         * Header used to track SPE Ring for telemetry
+         * Header used to track SPE Ring for telemetry.
          */
         static final String X_MS_CLITELEM = "x-ms-clitelem";
     }

--- a/adal/src/main/java/com/microsoft/aad/adal/AuthorityValidationMetadataCache.java
+++ b/adal/src/main/java/com/microsoft/aad/adal/AuthorityValidationMetadataCache.java
@@ -55,6 +55,10 @@ final class AuthorityValidationMetadataCache {
 
     private static ConcurrentMap<String, InstanceDiscoveryMetadata> sAadAuthorityHostMetadata = new ConcurrentHashMap<>();
 
+    private AuthorityValidationMetadataCache() {
+        // Utility class, no public constructor
+    }
+
     static boolean containsAuthorityHost(final URL authorityUrl) {
         return sAadAuthorityHostMetadata.containsKey(authorityUrl.getHost().toLowerCase(Locale.US));
     }

--- a/adal/src/main/java/com/microsoft/aad/adal/BrokerProxy.java
+++ b/adal/src/main/java/com/microsoft/aad/adal/BrokerProxy.java
@@ -363,7 +363,7 @@ class BrokerProxy implements IBrokerProxy {
             } catch (final AuthenticatorException e) {
                 // Error code BROKER_AUTHENTICATOR_ERROR_GETAUTHTOKEN will be thrown if there was an error
                 // communicating with the authenticator or if the authenticator returned an invalid response.
-                if (!StringExtensions.isNullOrBlank(e.getMessage()) && e.getMessage().contains(INVALID_GRANT)){
+                if (!StringExtensions.isNullOrBlank(e.getMessage()) && e.getMessage().contains(INVALID_GRANT)) {
                     Logger.e(TAG, AUTHENTICATOR_CANCELS_REQUEST,
                             "Acquire token failed with 'invalid grant' error, cannot proceed with silent request.",
                             ADALError.AUTH_REFRESH_FAILED_PROMPT_NOT_ALLOWED);
@@ -462,7 +462,7 @@ class BrokerProxy implements IBrokerProxy {
                     if (msg.contains(ADALError.NO_NETWORK_CONNECTION_POWER_OPTIMIZATION.getDescription())) {
                         adalErrorCode = ADALError.NO_NETWORK_CONNECTION_POWER_OPTIMIZATION;
                         break;
-                    } else if (msg.contains(ADALError.DEVICE_CONNECTION_IS_NOT_AVAILABLE.getDescription())){
+                    } else if (msg.contains(ADALError.DEVICE_CONNECTION_IS_NOT_AVAILABLE.getDescription())) {
                         adalErrorCode = ADALError.DEVICE_CONNECTION_IS_NOT_AVAILABLE;
                         break;
                     } else {

--- a/adal/src/main/java/com/microsoft/aad/adal/DRSMetadata.java
+++ b/adal/src/main/java/com/microsoft/aad/adal/DRSMetadata.java
@@ -34,7 +34,7 @@ final class DRSMetadata {
     private IdentityProviderService mIdentityProviderService;
 
     /**
-     * No args constructor for use in serialization for Gson to prevent usage of sun.misc.Unsafe
+     * No args constructor for use in serialization for Gson to prevent usage of sun.misc.Unsafe.
      */
     @SuppressWarnings("unused")
     DRSMetadata() {

--- a/adal/src/main/java/com/microsoft/aad/adal/Discovery.java
+++ b/adal/src/main/java/com/microsoft/aad/adal/Discovery.java
@@ -66,7 +66,7 @@ final class Discovery {
     /**
      * {@link ReentrantLock} for making sure there is only one instance discovery request sent out at a time.
      */
-    private volatile static ReentrantLock sInstanceDiscoveryNetworkRequestLock;
+    private static volatile ReentrantLock sInstanceDiscoveryNetworkRequestLock;
 
     /**
      * Sync set of valid hosts to skip query to server if host was verified

--- a/adal/src/main/java/com/microsoft/aad/adal/IdentityProviderService.java
+++ b/adal/src/main/java/com/microsoft/aad/adal/IdentityProviderService.java
@@ -36,7 +36,7 @@ final class IdentityProviderService {
     private String mPassiveAuthEndpoint;
 
     /**
-     * No args constructor for use in serialization for Gson to prevent usage of sun.misc.Unsafe
+     * No args constructor for use in serialization for Gson to prevent usage of sun.misc.Unsafe.
      */
     @SuppressWarnings("unused")
     IdentityProviderService() {

--- a/adal/src/main/java/com/microsoft/aad/adal/JWSBuilder.java
+++ b/adal/src/main/java/com/microsoft/aad/adal/JWSBuilder.java
@@ -59,7 +59,7 @@ class JWSBuilder implements IJWSBuilder {
     /**
      * Payload for JWS.
      */
-    class Claims {
+    final class Claims {
         @SerializedName("aud")
         private String mAudience;
 
@@ -70,7 +70,7 @@ class JWSBuilder implements IJWSBuilder {
         private String mNonce;
 
         /**
-         * No args constructor for use in serialization for Gson to prevent usage of sun.misc.Unsafe
+         * No args constructor for use in serialization for Gson to prevent usage of sun.misc.Unsafe.
          */
         @SuppressWarnings("unused")
         private Claims() {
@@ -80,7 +80,7 @@ class JWSBuilder implements IJWSBuilder {
     /**
      * Header that includes algorithm, type, thumbprint, keys, and keyid.
      */
-    class JwsHeader {
+    final class JwsHeader {
         @SerializedName("alg")
         private String mAlgorithm;
 
@@ -91,7 +91,7 @@ class JWSBuilder implements IJWSBuilder {
         private String[] mCert;
 
         /**
-         * No args constructor for use in serialization for Gson to prevent usage of sun.misc.Unsafe
+         * No args constructor for use in serialization for Gson to prevent usage of sun.misc.Unsafe.
          */
         @SuppressWarnings("unused")
         private JwsHeader() {

--- a/adal/src/main/java/com/microsoft/aad/adal/Link.java
+++ b/adal/src/main/java/com/microsoft/aad/adal/Link.java
@@ -45,7 +45,7 @@ final class Link {
     private String mHref;
 
     /**
-     * No args constructor for use in serialization for Gson to prevent usage of sun.misc.Unsafe
+     * No args constructor for use in serialization for Gson to prevent usage of sun.misc.Unsafe.
      */
     @SuppressWarnings("unused")
     Link() {

--- a/adal/src/main/java/com/microsoft/aad/adal/PowerManagerWrapper.java
+++ b/adal/src/main/java/com/microsoft/aad/adal/PowerManagerWrapper.java
@@ -32,7 +32,12 @@ import android.os.PowerManager;
  */
 
 public class PowerManagerWrapper {
-    static PowerManagerWrapper sInstance;
+
+    private static PowerManagerWrapper sInstance;
+
+    static void setInstance(final PowerManagerWrapper wrapper) {
+        sInstance = wrapper;
+    }
 
     /**
      * Singleton implementation for PowerManagerWrapper.
@@ -47,7 +52,7 @@ public class PowerManagerWrapper {
 
     /**
      * Wrap the calling to method isDeviceIdleMode() of final class PowerManager.
-     * @param connectionContext
+     * @param connectionContext Context used to query if app is in idle mode
      * @return true if the device is in doze/idle mode
      */
     @TargetApi(Build.VERSION_CODES.M)
@@ -57,7 +62,7 @@ public class PowerManagerWrapper {
 
     /**
      * Wrap the calling to method isIgnoringBatteryOptimizations() of final class PowerManager.
-     * @param connectionContext
+     * @param connectionContext Context used to query if app is ignoring battery optimizations
      * @return true if the given application package name is on the device's power whitelist
      */
     @TargetApi(Build.VERSION_CODES.M)

--- a/adal/src/main/java/com/microsoft/aad/adal/SSOStateSerializer.java
+++ b/adal/src/main/java/com/microsoft/aad/adal/SSOStateSerializer.java
@@ -63,7 +63,7 @@ final class SSOStateSerializer {
             .create();
 
     /**
-     * No args constructor for use in serialization for Gson to prevent usage of sun.misc.Unsafe
+     * No args constructor for use in serialization for Gson to prevent usage of sun.misc.Unsafe.
      */
     @SuppressWarnings("unused")
     private SSOStateSerializer() {

--- a/adal/src/main/java/com/microsoft/aad/adal/TokenCacheAccessor.java
+++ b/adal/src/main/java/com/microsoft/aad/adal/TokenCacheAccessor.java
@@ -113,7 +113,7 @@ class TokenCacheAccessor {
 
         TokenCacheItem item =  mTokenCacheStore.getItem(cacheKey);
         // try all the alias
-        if (item == null ) {
+        if (item == null) {
            item = performAdditionalCacheLookup(resource, clientId, null, user, REGULAR_TOKEN_ENTRY);
         }
 
@@ -230,7 +230,7 @@ class TokenCacheAccessor {
     /**
      * Update token cache with returned auth result.
      */
-    void updateTokenCache(final String resource, final String clientId, final AuthenticationResult result) throws MalformedURLException{
+    void updateTokenCache(final String resource, final String clientId, final AuthenticationResult result) throws MalformedURLException {
         if (result == null || StringExtensions.isNullOrBlank(result.getAccessToken())) {
             return;
         }

--- a/adal/src/main/java/com/microsoft/aad/adal/UsageStatsManagerWrapper.java
+++ b/adal/src/main/java/com/microsoft/aad/adal/UsageStatsManagerWrapper.java
@@ -32,7 +32,12 @@ import android.os.Build;
  */
 
 public class UsageStatsManagerWrapper {
-    static UsageStatsManagerWrapper sInstance;
+
+    private static UsageStatsManagerWrapper sInstance;
+
+    static synchronized void setInstance(final UsageStatsManagerWrapper instance) {
+        sInstance = instance;
+    }
 
     /**
      * Singleton implementation for UsageStatsManagerWrapper.
@@ -47,7 +52,7 @@ public class UsageStatsManagerWrapper {
 
     /**
      * Wrap the final class function UsageStatsManager.isAppInactive(). And make the code testable.
-     * @param connectionContext
+     * @param connectionContext Context used to query app active state
      * @return true if the app is inactive
      */
     @TargetApi(Build.VERSION_CODES.M)

--- a/adal/src/main/java/com/microsoft/aad/adal/WebFingerMetadata.java
+++ b/adal/src/main/java/com/microsoft/aad/adal/WebFingerMetadata.java
@@ -47,7 +47,7 @@ final class WebFingerMetadata {
     private List<Link> mLinks;
 
     /**
-     * No args constructor for use in serialization for Gson to prevent usage of sun.misc.Unsafe
+     * No args constructor for use in serialization for Gson to prevent usage of sun.misc.Unsafe.
      */
     @SuppressWarnings("unused")
     WebFingerMetadata() {

--- a/adal/src/telemetry/java/com/microsoft/aad/adal/TelemetryUtils.java
+++ b/adal/src/telemetry/java/com/microsoft/aad/adal/TelemetryUtils.java
@@ -47,40 +47,40 @@ final class TelemetryUtils {
             return mVersion;
         }
 
-        void setVersion(String mVersion) {
-            this.mVersion = mVersion;
+        void setVersion(String version) {
+            this.mVersion = version;
         }
 
         String getServerErrorCode() {
             return mServerErrorCode;
         }
 
-        void setServerErrorCode(String mServerErrorCode) {
-            this.mServerErrorCode = mServerErrorCode;
+        void setServerErrorCode(String serverErrorCode) {
+            this.mServerErrorCode = serverErrorCode;
         }
 
         String getServerSubErrorCode() {
             return mServerSubErrorCode;
         }
 
-        void setServerSubErrorCode(String mServerSubErrorCode) {
-            this.mServerSubErrorCode = mServerSubErrorCode;
+        void setServerSubErrorCode(String serverSubErrorCode) {
+            this.mServerSubErrorCode = serverSubErrorCode;
         }
 
         String getRefreshTokenAge() {
             return mRefreshTokenAge;
         }
 
-        void setRefreshTokenAge(String mRefreshTokenAge) {
-            this.mRefreshTokenAge = mRefreshTokenAge;
+        void setRefreshTokenAge(String refreshTokenAge) {
+            this.mRefreshTokenAge = refreshTokenAge;
         }
 
         String getSpeRing() {
             return mSpeRing;
         }
 
-        void setSpeRing(String mSpeRing) {
-            this.mSpeRing = mSpeRing;
+        void setSpeRing(String speRing) {
+            this.mSpeRing = speRing;
         }
     }
 

